Rohit/PostgresAssignment/src/controller/result.ts

```diff
@@ -0,0 +1,91 @@
+import { Context } from "vm";
+import { resultSchema } from "../validation/schema";
+import { updateResultSchema } from "../validation/schema";
+const resultController = require("../services/result");
+
+interface IResultInfo {
+  studentid: number;
+  class_Id: number;
+  subject_Id: number;
+  marks: number;
+}
+
+interface IUpdateResult {
+  studentid: number;
+  subject_Id: number;
+  marks: number;
+}
+
+exports.getResultData = async (ctx: Context) => {
+  try {
+    let [rows]: Array<{ rows: IResultInfo }> = [];
+    rows = await resultController.get_result();
+
+    ctx.response.status = 200;
+    ctx.response.type = "application/json";
+    ctx.body = rows.rows;
+  } catch (err) {
+    ctx.response.status = 400;
```

Comment:
common mistake... correct it in all the places...

Reply:
okk sir